client/stylesheets/_responsive.scss

```diff
@@ -0,0 +1,105 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Sass Document For Adding Responsitivity 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+@media (min-width: 1200px) {
+    .oneDiv, .onediv {
+        max-width: 1024px !important;
+    }
+}
+
+@media (max-width: 1200px) {
+    .oneDiv, .onediv {
+        width: 90% !important;
+    }
+}
+
+@media (max-width: 1065px) {
+    .homeCardLi {
+    	width: 19.5%;
+    }
+}
+
+@media (max-width: 1000px) {
+    .homeCardLi {
+    	width: 23%;
+    }
+}
+
+@media (max-width: 950px) {
+    .homeCardLi {
+    	width: 25%;
+    }
+}
+
+@media (max-width: 850px) {
+    .homeCardLi {
+    	width: 26%;
+    }
+}
+
+@media (max-width: 768px) {
+    .homeCardLi {
+    	width: 32%;
+    }
+}
+
+@media (max-width: 700px) {
+    .homeCardLi {
+    	width: 35%;
+    }
+}
+
+@media (max-width: 650px) {
+    .homeCardLi {
+    	width: 38%;
+    }
+}
+
+@media (max-width: 600px) {
+    .homeCardLi {
+    	width: 40%;
+    }
+}
+
+@media (max-width: 520px) {
+    .homeCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `homeCardLi` should be written in lowercase with hyphens